behat/features/bootstrap/FeatureContext.php

```diff
@@ -462,4 +462,78 @@ protected function getNodeByTitleAndBundle($title, $bundle) {
     $nid = $this->getNodeIdByTitleAndBundle($bundle, $title);
     return node_load($nid);
   }
+
+  /**
+   * @When I disable CI Build  :title
+   */
+  public function iDisableCiBuild($title) {
+    $node = $this->getNodeByTitleAndBundle($title, 'ci_build');
+    $wrapper = entity_metadata_wrapper('node', $node);
+    $wrapper->field_ci_build_enabled->set(FALSE);
+    $wrapper->save();
+  }
+
+  /**
+   * @When I enable CI Build  :title
+   */
+  public function iEnableCiBuild($title) {
+    $node = $this->getNodeByTitleAndBundle($title, 'ci_build');
+    $wrapper = entity_metadata_wrapper('node', $node);
+    $wrapper->field_ci_build_enabled->set(TRUE);
+    $wrapper->save();
+  }
+
+  /**
+   * @Then The CI Build item for build :title should be removed
+   */
+  public function theCiBuildItemForBuildShouldBeRemoved($title) {
+    $node = $this->getNodeByTitleAndBundle($title, 'ci_build');
+    $items = $this->getCiBuildItemsByStatus($node, 'queue');
+    if ($items) {
+      $error_message = format_string('The CI Build queue items for CI build "@title" are not removed',
+        array('@title' => $title));
+      throw new \Exception($error_message);
+    }
+  }
+
+  /**
+   * @Then The CI Build item for build :title should be created
+   */
+  public function theCiBuildItemForBuildShouldBeCreated($title) {
+    $node = $this->getNodeByTitleAndBundle($title, 'ci_build');
+    $items = $this->getCiBuildItemsByStatus($node, 'queue');
+    if (!$items) {
+      $error_message = format_string('The CI Build queue item for CI build "@title" is not created',
+        array('@title' => $title));
+      throw new \Exception($error_message);
+    }
+  }
+
+  /**
+   * Get all messages with certain status of certain CI Build.
+   *
+   * @param $node
+   *  The CI Build node.
+   * @param $status
+   *  Status field value. Defaults to "queue".
+   *
+   * @return array
+   *  Return array of messages IDs.
+   */
+  public function getCiBuildItemsByStatus($node, $status ='queue') {
```

Comment:
@amitaibu Is there a way to re-use the function from the `shoov_ci_build` module and not to duplicate the code?